app/views/events/participants.html.erb

```diff
@@ -41,24 +45,43 @@
             <span class="text-danger"><%= t '.unavailable' %></span>
           <% end %>
         </td>
+        <td>
+          <%= render 'colorselect', participant_group: @event.participant_group_for(participant) %>
+        </td>
         <% if can? :print_agreement_letters, Event and @has_agreement_letters %>
           <td>
-            <%= check_box_tag('selected_participants[]', participant.id, false, class: 'participant_selection') %>
+            <%= check_box_tag('selected_participants[]', participant.id, false, { class: 'participant_selection' }) %>
```

Comment:
Unless you need several html attributes, which in the end I didn't, so you're right ;)